### ISSUE TITLE
[BUG] Failing script in the ProblemChild integration package

### ISSUE THREAD

**Issue by ajosh0504** [elastic/integrations#3723]
While testing the ProblemChild integration package in an Elastic Cloud cluster, we found that Windows Endpoint Process events were not getting enriched with predictions from the ProblemChild model, with the following error:

<img width="914" alt="Screen Shot 2022-07-14 at 12 01 37 PM" src="https://user-images.githubusercontent.com/30438249/179062569-73e67bba-a599-448a-aa2e-c4693a594678.png">

Steps to reproduce:
1) Install the ProblemChild integration package in your Elastic Cloud cluster
2) Add the ProblemChild ingest pipeline to the existing ingest pipeline for the`logs-endpoint.events.process-*` indices
3) Ingest a Windows Process test document into the `logs-endpoint.events.process-*` indices
4) The above error appears in the `error.message` field, in the ingested document.

cc: @cojoe01

**Comment by ajosh0504** [comment]
After some debugging, I narrowed the error down to a failing script in the ProblemChild inference pipeline.

This is the first script in the sequence of processors in the inference pipeline. The way the signer information is being extracted in the script is what causes the error. `process.Ext.code_signature` is an array, and hence the contents of it can't be referenced by the `containsKey` method as done by the script:

```
if (ctx.containsKey('process') && ctx['process'].containsKey('Ext') && ctx['process']['Ext'].containsKey('code_signature') && ctx['process']['Ext']['code_signature'].containsKey('exists')) {
  ctx.feature_process_signed = ctx['process']['Ext']['code_signature']['exists']
}
```

The contents of the array, however, are also available as individual fields in the documents, such as `process.code_signature.exists`, `process.code_signature.trusted` etc. We can use these fields to extract the required features, instead of the `process.Ext.code_signature` field.

**Comment by szeitlin** [comment]
@ajosh0504 what's the priority on fixing this? is it still in progress, or on hold?

**Comment by ajosh0504** [comment]
It's in progress in the PR linked above ^. Currently trying to test the package with the updates.

**Comment by szeitlin** [comment]
Thanks, somehow I missed that (I blame end-of-week bleary eyes)

**Comment by ajosh0504** [comment]
Still waiting to merge the PR- will update the rule tags once we reach a consensus, then merge.

**Comment by szeitlin** [comment]
@ajosh0504 how long have you been waiting on responses to the last couple of questions/requests for review? If you don't get answers by Monday, let's ping those folks again directly and ask so we can close this out and move on.
